force-app/main/default/profiles/Sales Director %E2%80%93 APAC.profile-meta.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Profile xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Profile is not pushed properly  admin profile and field level security permissions information is not available here.